composer.json

```diff
@@ -37,6 +37,7 @@
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/orm": "^2.7",
         "ezyang/htmlpurifier": "dev-master as v4.14.0",
+        "fpt/autoload": "dev-main",
```

Comment:
This will, of course, have to be brought back into the PrestaShop organization before this PR can be merged.